app/package.json

```diff
@@ -29,7 +29,7 @@
     "react-error-overlay": "^6.0.9",
     "tailwindcss": "^3.3.3",
     "typescript": "^5.2.2",
-    "vite": "^3.0.4"
+    "vite": "^3.2.11"
```

Comment:
As I get it - it broke the build. Let's not touch it while it's working properly 🙂